app/src/evm/evm_utils.c

```diff
@@ -169,7 +169,14 @@ parser_error_t printBigIntFixedPoint(const uint8_t *number, uint16_t number_len,
         return parser_unexpected_value;
     }
 
-    fpstr_to_str(overlapped.output, sizeof(overlapped.output), bignum, decimals);
+    if (fpstr_to_str(overlapped.output, sizeof(overlapped.output), bignum, decimals)) {
+        return parser_unexpected_value;
+    }
+
+    if (z_str3join(overlapped.output, sizeof(overlapped.output), "SEI ", NULL) != zxerr_ok) {
```

Comment:
This could be a constant, right?

Reply:
Any reason why we set the ticket before the amount? I think we usually do the other way around.

Reply:
Sure, it can be. You are right there is now need to print it before, the non evm part of the app prints after, so I will change this to keep it constant trough tout the app.